examples/cms-plasmic/pages/[[...catchall]].tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react'
+import {
+  PlasmicComponent,
+  ComponentRenderData,
+  PlasmicRootProvider,
+  extractPlasmicQueryData,
+} from '@plasmicapp/loader-nextjs'
+import { GetStaticPaths, GetStaticProps } from 'next'
+import Error from 'next/error'
+import { PLASMIC, PREVIEW_PLASMIC } from '../plasmic-init'
```

Comment:
I think it would be useful to distinguish between `import` and `import type` here. What do you think?